packages/model-viewer/src/features/scene-graph/material.ts

```diff
@@ -110,13 +110,13 @@ export class Material extends ThreeDOMElement implements MaterialInterface {
         aoMap: verifyAoMap,
         emissiveMap: verifyEmissiveMap
       } = gltfMaterial;
-      if (verifyNormalMap !== normalMap) {
+      if (verifyNormalMap != normalMap) {
```

Comment:
Ah, I see why you had `!==` here. I don't love this, but I think it's fine for now. Let's see if we can get rid of these warnings entirely down the road a little.